capirca/lib/paloaltofw.py

```diff
@@ -770,6 +766,8 @@ def _BuildPort(self, ports):
   def __str__(self):
     """Render the output of the PaloAltoFirewall policy into config."""
 
+    add_any_ipv4 = False
```

Comment:
A comment here indicating that this is used in the case of both inet/inet6 (since it's counterintuitive).